easybuild/easyconfigs/m/MXNet/MXNet-1.9.1-foss-2022a.eb

```diff
@@ -0,0 +1,67 @@
+name = 'MXNet'
+version = '1.9.1'
+
+homepage = 'https://mxnet.io/'
+description = """Flexible and Efficient Library for Deep Learning"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'cstd': 'c++14', 'opt': True, 'pic': True, 'openmp': True, 'extra_cflags': '-lflexiblas'}
```

Comment:
To be still need extra manual cflags here despite including the patch to their cmakelists to support flexiblas?

Reply:
yep, won't build without it

Reply:
you're right, this doesn't have to be in patch, removing...

Reply:
well, i don't think there is any advantage to modifying the patch. i'm mostly curious as to where it fails

Reply:
@Micket I think it's working now as is, but if you want, we could move the `-lflexiblas` option from `extra_cflags` in easyconfig to the easyblock somehow, however, I am not sure how to modify toolchainopts in easyblock...if you think it might be better that way, maybe drop me a hint about how to do that?